doc/reference/configuration/cfg_replication.rst

```diff
@@ -428,12 +428,30 @@
     This option tells how many replicas should confirm the receipt of a
     synchronous transaction before it can finish its commit.
 
+    Since version :doc:`2.5.3 </release/2.5.3>`,
+    the option supports dynamic evaluation of the quorum number.
+    That is, the number of quorum can be specified not as a constant number, but as a function instead.
+    In this case, the option returns the formula evaluated.
+    The result is treated as an integer number.
+    Once any replicas are added or removed, the expression is re-evaluated automatically.
+
+    For example,
+
+    ..  code-block:: lua
+
+        box.cfg{replication_synchro_quorum = "(N/2)+1"}
+
+    Where `N` is a current number of registered replicas in a cluster.
+    Keep in mind that the example above represents a canonical quorum definition.
+    The formula ``at least 50% of the cluster size + 1`` guarantees data reliability.
+    Using a value less than the canonical one might lead to unexpected results,
+    including a :ref:`split-brain <repl_leader_elect_splitbrain>`.
+
     Since version :doc:`2.10.0 </release/2.10.0>`, this option
     does not account for anonymous replicas.
 
-    It is 1 by default, so synchronous transactions work like asynchronous when
-    not configured. 1 means successful WAL write on master is enough for
-    commit.
+    The default value for this parameter is `1`, so synchronous transactions work like asynchronous when not configured.
```

Comment:
By the way, since 2.10.0 the default value is "N / 2 + 1". Let's reflect it here.
I couldn't find the corresponding doc request, must've forgotten to add it. But the corresponding commit is
https://github.com/tarantool/tarantool/commit/f0eb2ca3f9

Reply:
By the way (2), such behavior variations should be described like so:

> Since A.B.C: <...>.
>
> Since X.Y.Z: <...>.